README.md

```diff
@@ -110,11 +113,16 @@ pip install mkdocs mkdocs-material
 
 ### Start the local server
 
-Run `mkdocs serve` to start a local server on `http://127.0.0.1:8000`. Now, whenever
-you make changes to any file, the server will automatically reload and display the new
-version.
+Run `mkdocs serve` to start a local server on `http://127.0.0.1:8000`. Now, whenever you make
+changes to any file, the server will automatically reload and display your edited version.
 
 ### Build the docs
 
 To build the `aimoes-docs` locally, use `mkdocs build`. You can now open the documentation
 in your browser from the newly generated `<your-aimeos-docs-clone-folder>/docs`´s index.html.
+
+### Troubleshooting
+
+Due to some configuration limits of mkdocs, the command line will print out a 404
```

Comment:
That's not entirely correct. Only `mkdocs serve` will tell you that `version.js` isn't found